core/src/main/scala/better/files/File.scala

```diff
@@ -690,7 +690,6 @@ class File private(val path: Path)(implicit val fileSystem: FileSystem = path.ge
     */
   def copyTo(destination: File, overwrite: Boolean = false)(implicit copyOptions: File.CopyOptions = File.CopyOptions(overwrite)): destination.type = {
     if (isDirectory) {//TODO: maxDepth?
-      if (overwrite) destination.delete(swallowIOExceptions = true)
```

Comment:
Thanks for your review and suggestion! :D I will try to add 4 test cases in the weekend then I could have more idea. :)